app/routes.js

```diff
@@ -166,7 +166,6 @@ export const requireNotVerified = (api, store) => (nextState, replace, cb) => {
           return cb();
         }
         throw new Error('Error getting user at /email-verification');
-        return cb();
```

Comment:
I took it out as it would never have been called because of the error thrown right before it (my js linter flagged it, so I figured I'd might as well clean it up)